testsuite/setup.py

```diff
@@ -77,122 +72,16 @@ def run(self):
-                         'data/tprs/*/*.tpr',
-                         'data/*.gro', 'data/*.xtc', 'data/*.trr', 'data/*npy',
-                         'data/*.crd', 'data/*.xyz',
-                         'data/Amber/*.bz2',
-                         'data/Amber/*.prmtop', 'data/Amber/*.top',
-                         'data/Amber/*.parm7',
-                         'data/Amber/*.rst7',
-                         'data/Amber/*.trj', 'data/Amber/*.mdcrd',
-                         'data/Amber/*.ncdf', 'data/Amber/*.nc',
-                         'data/Amber/*.inpcrd',
-                         'data/gromos11/*.gz', 'data/gromos11/*.trc',
-                         'data/*.pqr', 'data/*.pdbqt', 'data/*.bz2', 'data/*.gz',
-                         'data/*.ent',
-                         'data/*.fasta',
-                         'data/*.dat',
-                         'data/*.dms',
-                         'data/merge/2zmm/*.pdb',
-                         'data/*.trz',
-                         'data/mol2/*.mol2',
-                         'data/contacts/*.gro.bz2', 'data/contacts/*.dat',
-                         'data/capping/*.gro', 'data/capping/*.pdb',
-                         'data/lammps/*',
-                         'data/gms/*.xyz', 'data/gms/*.gms',
-                         'data/gms/*.gms.gz',
-                         'data/*.inpcrd',
-                         'data/dlpoly/CONFIG*',
-                         'data/dlpoly/HISTORY*',
-                         'data/*.xml',
-                         'data/coordinates/*',
-                         'data/*xvg',
-                         'data/*.mmtf', 'data/*.mmtf.gz',
-                         'data/analysis/*',
-                         'data/*.gsd',
-                         'data/windows/*',
-                         'data/*.itp', "data/gromacs/gromos54a7_edited.ff/*",
-                         'data/*.coor',
-                         'data/*.h5md',
-                         'data/*.in',
-                         'data/*.top',
-                         'data/*.sdf',
-                         'data/*.edr',
-                         'data/*.tng',
-                         'data/*.pdbx',
-                         'data/*.txt',
-                        ],
-          },
-          python_requires='>=3.9',
-          install_requires=[
-              'MDAnalysis=={0!s}'.format(RELEASE),  # same as this release!
-              'pytest>=3.3.0', # Raised to 3.3.0 due to Issue 2329
-              'hypothesis',
-          ],
-          # had 'KeyError' as zipped egg (2MB savings are not worth the
-          # trouble)
-          zip_safe=False,
-          cmdclass={'sdist': MDA_SDist},
-          )
+    setup(
+        version=RELEASE,
+        install_requires=[
+            'MDAnalysis=={0!s}'.format(RELEASE),  # same as this release!
+            'pytest>=3.3.0', # Raised to 3.3.0 due to Issue 2329
+            'hypothesis',
+        ],
```

Comment:
Does this remains here because of the requirement that the version match MDAnalysis?

Reply:
Yes, anything with dynamic matching must be in setup.py